### ISSUE TITLE
Is it possible to pull location/geo data from another users post?

### ISSUE THREAD

**Issue by zachasman** [adw0rd/instagrapi#773]
### Compensate with donations ($20 and more) for the time I spent to implement new functionality or send me your [Pull Request](https://github.com/adw0rd/instagrapi/pulls) with your implementation of the new functionality

[![Donate](https://www.buymeacoffee.com/assets/img/custom_images/yellow_img.png)](https://www.buymeacoffee.com/adw0rd)

* Bitcoin: 3KHgnvNNWio6wyQ71cgNeBQojXStxzkzfd
* Ethereum: 0x6e644c6b975B47801072b07a9a57DDfEC10DE74F
* https://paypal.me/adw0rd
* For Russia: СБП, Qiwi и тп: **+79118107609**

**Is your feature request related to a problem? Please describe.**
Yes, I'm trying to pull the location from a users media.

**Describe the solution you'd like**
The ability to pull a users geo data from their posts (but not from my posts, rather people I follow)

**Describe alternatives you've considered**
Using various endpoints from this package

**Additional context**
Is this a thing that can still be done? I know Instagram has changed a bunch of stuff from their API in the last year or so.

**Comment by subzeroid** [comment]
Yes, but only when Instagram returns a location object for that media, and exact lat/lng is not guaranteed for every post/place.

Use `media.location` from `media_info(...)` or from media returned by `user_medias(...)` / location feeds:

```python
media = cl.media_info(media_pk)
if media.location:
    print(media.location.pk, media.location.name, media.location.lat, media.location.lng)
```

If you only get a place id/name, you can try to enrich it:

```python
location = cl.location_info(media.location.pk)
```

or search around known coordinates / by name:

```python
places = cl.location_search(lat, lng)
places_by_name = cl.location_search_name(media.location.name)
```

Current verification: a recent public media from a location feed returned `media.location` with a place id and name. `location_info()` for that place did not return lat/lng in that response, while `location_search(...)` does return lat/lng for search results. So the library can expose the geo/place data Instagram sends, but it cannot reconstruct hidden coordinates when Instagram omits them from the media/place response.

Docs:
Media: https://subzeroid.github.io/instagrapi/usage-guide/media.html
Location: https://subzeroid.github.io/instagrapi/usage-guide/location.html

Closing this as answered/current behavior rather than a missing API method.
